admin_manual/configuration/server/config_sample_php_parameters.rst

```diff
@@ -722,32 +722,38 @@ The default value is ``ownCloud``.
 			'shared_secret' => '57b58edb6637fe3059b3595cf9c41b9',
 			'users' => ['user1'],
 			'apps' => ['files_texteditor'],
-			'logfile' => '/tmp/test.log'
+			'logfile' => '/tmp/test1.log'
 	        ],
 	        [
 			'shared_secret' => '57b58edb6637fe3059b3595cf9c41b9',
-			'users' => ['user1'],
+			'users' => ['user2'],
 			'apps' => ['gallery'],
-			'logfile' => '/tmp/gallery.log'
+			'logfile' => '/tmp/test2.log'
 	        ],
 	],
 
-Log condition for log level increase based on conditions. Once one of these
-conditions is met, the required log level is set to debug. This allows to
-debug specific requests, users or apps
+Log condition for log level increase based on conditions. 
```

Comment:
seems like "increase" should be "increases".

Reply:
This is the original wording.
Imho you can let increase (because it increases the level), we also can take increasement.
I am open, just let me know and I correct it.